### PR TITLE
fix: error while setting up local development

### PR DESCRIPTION
**Pull request by anasaijaz**
# Description

docs: added instruction to setup environment
add: Error message if APP_ENV is undefined

# Issue
Environment error while setting up in local

# Checklist:

- [x] My commit message follows the Conventional Commits specification
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have tested my code
- [x] My changes generate no new warnings
- [x] My PR is rebased off the most recent `main` or appropriate feature branch
- [x] My PR is opened against the `main` or appropriate feature branch

# Alert Reviewers

@codynhat @gravenp

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,16 @@
 ```
 yarn
 ```
+## Environment
+Windows
+```
+set APP_ENV = testing
```
Comment (codynhat) [inline README.md:10]: The `APP_ENV` variable will select a particular `.env` file to use. The best default value is `APP_ENV=testnet`.

**Review by codynhat — CHANGES_REQUESTED** [review]
Thanks for the PR @anasaijaz!

File: README.md (hunk 1 above)
Comment (anasaijaz) [reply README.md]: Oh really sorry! missed that

Fixed this in a recent commit

**Comment by anasaijaz** [comment]
Hi @codynhat I have fixed the default value issue. Can you please merge my request

**Comment by codynhat** [comment]
Sorry, looking at why the build is failing and then I will merge. Thanks!

**Pull request merged**